packages/nx-playwright/README.md

```diff
@@ -17,17 +17,25 @@ yarn playwright install --with-deps
 yarn nx e2e <APP-NAME>-e2e
 ```
 
-## Execution Flags
-
-`nx-playwright` has some flags that you can utilize at execution time
-
-- `--browser=BROWSER_TYPE`: allowed browser types being `chromium`, `firefox` or `webkit` (or an `all` type to execute against all 3 types)
-- `--format=FORMAT_TYPE`: this allows values such as `json` or `html`
-- `--headed`: launches the browsers in non-headless mode
-- `--skipServe`: skips the execution of a devServer
-- `--timeout=TIMEOUT_IN_MS`: adds a timeout for your tests
```

Comment:
This documentation can possible be re-located, or re-worded, but removing it makes the plugin less clear

i'd suggest adding an explanatory line like

    nx e2e my-app-e2e --browser=firefox --headed --skipServe

Reply:
These changes have been reverted for now - see latest comment